src/BlazorUI/Bit.BlazorUI/Components/Buttons/ButtonGroup/BitButtonGroup.scss

```
@@ -28,6 +29,49 @@
+    border-radius: 9999px;
+
+    &.bit-btg-vrt {
+        border-radius: $shp-border-radius;
+    }
+}
+
+.bit-btg-scr,
+.bit-btg-scb {
+    overflow-x: auto;
+    overflow-y: hidden;
+    // A swipe past either end scrolls the group only, never the page behind it.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)